redhat-developer/app-services-cli
# feat(login): add branding to login redirect page

This PR includes the new product branding into the login web page.

### Verification Steps


1. Run the `login` page
2. The login web page should include the new branding on the main body and as a favicon.

![image](https://user-images.githubusercontent.com/11743717/127519036-498f491a-fa01-4ac6-a72a-6d12d6027653.png)


### Type of change


### Checklist
